### PR TITLE
Edit tizen studio content

### PR DESCRIPTION
**Pull request by l10nbridge**
Performed basic editing, fixing language and structure.
Improved usability and readability.

### CODE REVIEW

File: docs/open-source-tizen/tizen-studio/example-web-certi-ext.md [hunk 1]
```diff
@@ -1,91 +1,85 @@
...
+     - `normalImage`: Plugin-relative path to the image to be displayed as the normal state for the creation button
+     - `selectedImage`: Plugin-relative path to the image to be displayed as the selected state for the creation button
+     - `hoverImage`: Plugin-relative path to the image to be displayed as the hover state for the creation button
+     - `handler`: Fully-qualified name of a handler class that implements `org.tizen.common.sign.ui.ICertificateCreationHandler`. Call the `execute()` method of this handler when the button is clicked.
+
+2. Implement the new plugin in the `plugin.xml` file:
+
+   ![Changes to plugin.xml](media/cert-ext.png)
+
+3. Add `org.tizen.common.sign.ui` into the `Require-bundle` attribute list:
```
Comment (l10nbridge) [inline docs/open-source-tizen/tizen-studio/example-web-certi-ext.md]: Where is this list located?

File: docs/open-source-tizen/tizen-studio/package-file-desc-guide.md [hunk 2]
```diff
@@ -70,38 +80,38 @@ build() {
...
     cp $SRCDIR/a $INSTALL_DIRA
 }
 ```
-#### **changelog** file
 
-- Included in package file (zip)
-- Update changelog when new package is deployed. <Version Up required>
+### `changelog` File
+
+This file is included in the package file (`.zip`). When a new package is deployed, the changelog is updated. &lt;Version Up is required&gt;
```
Comment (l10nbridge) [inline docs/open-source-tizen/tizen-studio/package-file-desc-guide.md]: What does the `<version up is required>` comment mean?

File: docs/open-source-tizen/tizen-studio/package-file-desc-guide.md [hunk 3]
```diff
@@ -70,38 +80,38 @@ build() {
...
 
 ```text
  * 1.0.12    // Same as pkginfo.manifest version
-- Titile, Desc ,,
+- Title, Desc ,,
 == Sujin Lee <lee.sujin@samsung.com> 20XX-XX-XX
```
Comment (l10nbridge) [inline docs/open-source-tizen/tizen-studio/package-file-desc-guide.md]: Are you sure you want to use the name of a real person in this example?

File: docs/open-source-tizen/tizen-studio/contribute-gerrit.md [hunk 4]
```diff
@@ -1,60 +1,53 @@
...
+    git add <file name>
+    git commit -m <update massage>
+    git log // To successfully push your code, you must include 'change-id' and 'Signed-off-by' lines.
+    git push origin <branch name>
+    ```
+
+    The change is listed in [Gerrit Code Review](https://review.tizen.org/gerrit).
+
+6. Request a code review.  
+    In Gerrit, open your change line and add the extension maintainer as a reviewer.
```
Comment (l10nbridge) [inline docs/open-source-tizen/tizen-studio/contribute-gerrit.md]: The text says to add the maintainer in Gerrit. However, the screenshot is from GitHub. So do you actually need to access the maintainer name first in GitHub, and then go to Gerrit and add it there?

File: docs/open-source-tizen/tizen-studio/local-build.md [hunk 5]
```diff
@@ -1,84 +1,76 @@
...
+   ## -f, --force           skip version comparison and push or pull packages by force.
+                            new packages will overwrite existing ones.
+   ```
 
-To create a snapshot, use the command `ts-cli push`
+   A `snapshots` folder is created in the specified location and a snapshot is created under it.
 
-```java
-$ ts-cli push -P <package file path|list> --lr /repository/tizen_studio
+   ![Snapshot creation results](media/snapshot-result.png)
```
Comment (l10nbridge) [inline docs/open-source-tizen/tizen-studio/local-build.md]: This file contains Korean characters. Can you take a new one without them?

File: docs/open-source-tizen/tizen-studio/local-build.md [hunk 6]
```diff
@@ -1,84 +1,76 @@
...
+   ```bash
+   $ ts-cli create-image -r /repository/tizen_studio -u http://download.tizen.org/sdk/tizenstudio/official -O MyImage
 
+   ## -r, --repository      local repository path in filesystem
+   ## -u, --url             base repository URL
+   ## -O, --output          image name
+   ```
 
-### Step 5 : Creating an Installation Image
+   ![Image creation results](media/image-result.png)
```
Comment (l10nbridge) [inline docs/open-source-tizen/tizen-studio/local-build.md]: This file contains Korean characters. Can you take a new one without them?

File: docs/open-source-tizen/tizen-studio/contribute-gerrit.md (hunk 4 above)
Comment (sujin0830) [reply docs/open-source-tizen/tizen-studio/contribute-gerrit.md]: I will remove the image and add it as text.

File: docs/open-source-tizen/tizen-studio/emulator-extension-guide.md [hunk 7]
```diff
@@ -1,142 +1,128 @@
...
+3. Build the code:
+   - In the case of `lib`, `ui`, and `cli` libraries, to build them all at once, enter the `ant` shell command at the top level of your project.
+   - In the case of devices:
+     - When building is complete, copy the built `libecp`, `emulator-control-panel.jar`, and `emulator-control-panel-cli.jar` files to `libs` in the devices project directory.
+     - To build the `mobile`, `wearable`, and `tv` profiles all at once, enter the `ant` shell command at the top level of your project. To build a specific profile only, you can specify, for example, `ant -f ECP-DEVICE/build_mobile.xml`.
+4. If you configure devices separately, you only need to push changes to the `emulator-control-panel-devices` repository.
+5. When building is complete, move the following files:
+    - `lib`, `ui`, and `cli` to `sdk/tools/emulator/bin`
+    - Device `xml` and `jar` files to `platforms/<version>/<profile>/emulator-resources/plugins`
+        - You must also rename the `.jar` file.
```
[the remark this answers is not in the document]
Comment (sujin0830) [reply docs/open-source-tizen/tizen-studio/emulator-extension-guide.md]: I will explain further.

File: docs/open-source-tizen/tizen-studio/example-web-certi-ext.md [hunk 8]
```diff
@@ -1,91 +1,85 @@
...
+To modify the content of the certificate type selection window of the Certificate Manager:
+
+1. Use the `org.tizen.common.sign.ui.certificateTypes` extension point:
+
+   - Identifier: `org.tizen.common.sign.ui.certificateTypes`
+   - Description: This extension point is used for registering the extension certificate type.
+   - Configuration:
+     ```xml
+     <!ELEMENT extension (type)>
+     <!ATTLIST extension
```
[the remark this answers is not in the document]
Comment (sujin0830) [reply docs/open-source-tizen/tizen-studio/example-web-certi-ext.md]: Extension development is the same as the Eclipse plug-in development.

File: docs/open-source-tizen/tizen-studio/emulator-extension-guide.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (sujin0830) [reply docs/open-source-tizen/tizen-studio/emulator-extension-guide.md]: Uploaded again. done.

File: docs/open-source-tizen/tizen-studio/emulator-extension-guide.md [hunk 9]
```diff
@@ -1,142 +1,128 @@
...
+   For example:  
+   ```
+   vm_name=w-0906-1 base_port=26100 platform_version=tizen-2.3.2 profile=wearable
+   ```
+   The platform version and profile are used in the directory names:
+   ```
+   <tizen-sdk>/platforms/<platform_version>/<profile>
+   ```
+4. In the run configuration, enter the following VM arguments:
+    - `-Ddevelop`, `sdk.info` file is found based on the SDK installation path specified in `FilePath.java`
```
[the remark this answers is not in the document]
Comment (sujin0830) [reply docs/open-source-tizen/tizen-studio/emulator-extension-guide.md]: Uploaded again. done.

File: docs/open-source-tizen/tizen-studio/emulator-extension-guide.md [hunk 10]
```diff
@@ -1,142 +1,128 @@
...
+    - For example: `emulator-control-panel/ECP-LIB`
+      - `ln -s ~/tizen-studio/sdk.info sdk.info`
+
+### Changes
```
[the remark this answers is not in the document]
Comment (sujin0830) [reply docs/open-source-tizen/tizen-studio/emulator-extension-guide.md]: Uploaded again. done.

File: docs/open-source-tizen/tizen-studio/emulator-extension-guide.md [hunk 11]
```diff
@@ -1,142 +1,128 @@
...
+
+- Use JavaFX to implement UI features. SWT is not compatible with the existing code (including XML).
+- Rename the plugin `ecp-plugin.xml` file to `ecp-plugin-<profile>-<version>.xml`.
+- Change the package structure:
+    - Separate the code to the `emulator-control-panel` and `emulator-control-panel-devices` Git repositories.
+    - Change the executable file under platforms to Tools.
+- Change the XML form.
```
[the remark this answers is not in the document]
Comment (sujin0830) [reply docs/open-source-tizen/tizen-studio/emulator-extension-guide.md]: Uploaded again. done.

File: docs/open-source-tizen/tizen-studio/web-extension-guide.md [hunk 12]
```diff
@@ -1,54 +1,57 @@
...
+  - `org.eclipse.ui.popupMenus`
+  - `org.eclipse.ui.newWizards`
+  - `org.eclipse.ui.editors`
+  - `org.eclipse.ui.propertyPages`
+  - `org.eclipse.ui.perspectiveExtensions`
+  - `org.eclipse.help.toc`
+
+  For an example, see [Modifying Eclipse Themes](example-web-theme-ext.md).
+
+If you have additional extension points, contact us.
```
[the remark this answers is not in the document]
Comment (sujin0830) [reply docs/open-source-tizen/tizen-studio/web-extension-guide.md]: Delete this sentence. Thank you.

**Comment by mijin85cho** [comment]
Merge this PR for further PR (To make another PR based one this) : Requested by Sujin Lee

**Pull request merged**